microsoft/react-native-macos
# Implementing AccessibilityInfo.DisplayOptions ex: isInvertColorsEnabled

#### Please select one of the following

## Summary

When the MacOS fork was made all the AccessibilityInfo functionality was stubbed out because the iOS Accessibility APIs are different from the MacOS APIs. I am implementing several AccessibilityInfo properties and EvenListeners. This will allow a application to tell if several Accessible Display Options are enabled, or if they change.

Specifically this PR implements AccessibilityInfo.isInvertColorsEnabled AccessibilityInfo.isReduceMotionEnabled and AccessibilityInfo.isReduceTransparencyEnabled.

## Changelog

[MacOS] [Added] - Implemented AccessibilityInfo.isInvertColorsEnabled, isReduceMotionEnabled, & isReduceTransparencyEnabled

## Test Plan
1. Run RNTester
2. Go to the accessibility page 
3. Scroll to the Display Options Enabled section (second to last)
4. Open Settings>Accessibility>Display
5. Toggle Invert Colors, Reduce Motion, and Reduce Transparency.
- You should see the text change back and forth between for example "Invert colors iss disabled" and "Invert colors is enabled"

![Screen Shot 2020-06-16 at 14 51 12](https://user-images.githubusercontent.com/65255457/84836072-ea37fd00-afe9-11ea-914c-f57fd5efb4bb.png)

![Screen Shot 2020-06-16 at 14 50 41](https://user-images.githubusercontent.com/65255457/84836087-f15f0b00-afe9-11ea-8f2c-5eb92902e6fd.png)


###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/455)
